### PR TITLE
RecommendationTitle: https://github.com/metanorma/metanorma-ogc/issue…

### PR DESCRIPTION
**Pull request by opoudjis**
…s/56

### CODE REVIEW

File: spec/asciidoctor/base_spec.rb [hunk 1]
```diff
@@ -1069,7 +1069,7 @@
            <substage>active</substage>
          </status>
          <copyright>
-           <from>2019</from>
+           <from>#{Date.today.year}</from>
```
[the remark this answers is not in the document]
Comment (ronaldtse) [reply spec/asciidoctor/base_spec.rb]: @opoudjis I think these need to be fixed.

**Comment by ronaldtse** [comment]
Build failing right now. I think the `Date.today` issues need to be fixed, hound is right.

**Comment by opoudjis** [comment]
The fail is because of isodoc CSS base class update, needs to wait for release.

File: spec/asciidoctor/base_spec.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (opoudjis) [reply spec/asciidoctor/base_spec.rb]: I think Hound needs to be fixed, so it does not demand Rails-specific methods. Now I have to drag activesupport in...

[the remark this answers is not in the document]
Comment (ronaldtse) [reply spec/asciidoctor/base_spec.rb]: Oh, please don't use activesupport...

[the remark this answers is not in the document]
Comment (ronaldtse) [reply spec/asciidoctor/base_spec.rb]: Let's fix Hound that it doesn't use Rails-specific recommendations. Thanks.

**Pull request merged**